pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.10</version>
+            <version>4.11</version>
```

Comment:
Are we intentionally trying to increase the version of Junit? That sounds like a different kind of change than the rest of whats done here. I think you want to be the version on line 8 instead.